test/test_crossinheritance.py

```diff
@@ -184,7 +184,7 @@ def sum_all(self, *args):
         assert d.sum_all(-7, -5)             == 1
         assert Base1.call_sum_all(d, -7, -5) == 1
 
-    @mark.xfail(run=IS_CLANG_REPL, reason="Crashes on cling when \
+    @mark.skipif(not IS_CLANG_REPL, reason="Crashes on cling when \
```

Comment:
```suggestion
    @mark.skipif(run=not IS_CLANG_REPL, reason="Crashes on cling when \
```

Don't we need something like that?

Reply:
Ok, I overlooked that we swap `xfail` with `skipif`.